pyproject.toml

```diff
@@ -1,10 +1,12 @@
 [build-system]
 build-backend = "mesonpy"
-backend-path = ['./vendored-meson/meson-python']
+#backend-path = ['./vendored-meson/meson-python']
 requires = [
     "Cython>=3.0",
     # All dependencies of the vendored meson-python (except for meson, because
     # we've got that vendored too - that's the point of this exercise).
+    'meson-python',
+    'vendored-meson/meson',
```

Comment:
This doesn't seem to work. Does it work offline? All the builds fail for this line.

Reply:
accidental commit, never mind that one.